infrastructure/pom.xml

```diff
@@ -24,28 +24,38 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
+
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-actuator</artifactId>
         </dependency>
+
+        <dependency>
```

Comment:
Why don't you also add the database settings in the application.yml?

Reply:
Spring add the default configuration, anyway we are not using a mongo repository. So we can add the needed configuration when we are using it.